libs/metrics/base.py

```diff
@@ -26,6 +26,7 @@ def parse_date_range(dates):
 
 
 class BaseMetric:
+    metric_name = None
```

Comment:
Why this attribute was added?  I don't find any usage of it

Reply:
Forgot to delete it here